src/main/java/com/ft/methodearticlemapper/resources/PostContentToTransformResource.java

```diff
@@ -83,19 +81,14 @@ private Content processRequest(String uuid, boolean preview, EomFile eomFile, St
         }
     }
 
-    private void validateUuid(String uuid, EomFile eomFile) {
+    private void validateUuid(String uuid) {
         if (uuid == null) {
             throw ClientError.status(400).context(null).reason(ErrorMessage.UUID_REQUIRED).exception();
         }
         try {
-
-            UUID resourceId = UUID.fromString(uuid);
-
-            if (!uuid.equals(eomFile.getUuid())) {
-                String errorMessage = String.format(ErrorMessage.CONFLICTING_UUID.toString(), uuid, eomFile.getUuid());
-                throw ClientError.status(409)
-                        .error(errorMessage)
-                        .exception(new MethodeContentInvalidException(resourceId, errorMessage));
+            UUID modelUUID = UUID.fromString(uuid);
```

Comment:
since we don't use the `modelUUID` for anything else, I'd write this as a one liner:
`if (!UUID.fromString(uuid).toString().equals(uuid))`